README.md

```diff
@@ -165,78 +184,566 @@ and then destroy them, all from the command line:
+
+#####`tags`
+*Optional* The tags to assign to the VPC. Accepts a 'key => value' hash of tags.
+
+#### Type: ec2_vpc_customer_gateway
+Type representing an AWS VPC customer gateways.
+
+#####`name`
+*Required* The name of the customer gateway.
+
+#####`ip_address`
+*Required* The IPv4 address for the customer gateway. Accepts a valid IPv4 address.
+
+#####`bgp_asn`
+*Required* The Autonomous System Numbers for the customer gateway.
+
+#####`tags`
+*Optional* The tags for the customer gateway. Accepts a 'key => value' hash of tags.
+
+#####`region`
+*Optional* The region in which to launch the customer gateway. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+
+#####`type`
+The type of customer gateway. The only currently supported value --- and the default --- is 'ipsec.1'.
+
+#### Type: ec2_vpc_dhcp_options
+
+Sets a DHCP option AWS VPC.
+
+#####`name`
+*Required* The name of the DHCP options set. Accepts a string.
+
+#####`tags`
+*Optional* Tags for the DHCP option set. Accepts a 'key => value' hash of tags.
+
+#####`region`
+*Optional* The region in which to assign the DHCP option set. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+
+#####`domain_name`
+*Optional* The domain name for the DHCP options. Accepts any valid domain. 
+
+#####`domain_name_servers` 
+*Optional* A list of domain name servers to use for the DHCP options set. Accepts an array of domain server names. 
+
+#####`ntp_servers
+*Optional* A list of NTP servers to use for the DHCP options set. Accepts an array of NTP server names.
+
+#####`netbios_name_servers`
+*Optional* A list of netbios name servers to use for the DHCP options set. Accepts an array.
+
+#####`netbios_node_type`
+*Optional* The netbios node type. Valid values are '1', '2', '4', '8'. Defaults to '2'.
+
+
+#### Type: ec2_vpc_internet_gateway
+
+Sets up an EC2 VPC Internet Gateway.
+
+#####`name`
+*Required* The name of the internet gateway.
+ 
+#####`tags`
+*Optional* Tags to assign to the internet gateway. Accepts a 'key => value' hash of tags.
+
+#####`region`
+*Optional* The region in which to launch the internet gateway. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+
+#####`vpc`
+*Optional* The vpc to assign this internet gateway to.
+
+
+#### Type: ec2_vpc_routetable
+
+Sets up a VPC route table.
+
+#####`name`
+*Required* The name of the route table.
+
+#####`vpc`
+*Optional* VPC to assign the route table to.
+
+#####`region`
+*Optional* The region in which to launch the route table. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+
+#####`routes`
+*Optional* Individual routes for the routing table. Accepts an array of 'destination_cidr_block' and 'gateway' values:
+ 
+~~~
```

Comment:
Good question! Docs has been moving to ~~~ as it renders properly in our docs; my understanding is that the language specific highlights do not. I think there are good reasons to use the language fences, but they may not work for us everywhere. I'll have some conversations about this tomorrow.